woodwork/tests/utils/test_utils.py

```diff
@@ -617,3 +618,20 @@ def test_callback_caller_no_callback():
     caller.update(1)
 
     assert caller.current_progress == 0
+
+
+def test_concat_dataframe_or_series():
```

Comment:
Nit: can we add a test for dataframes as well to ensure that works. Also, can we check the ordering of the data and/or index of the result is what we expect?

Reply:
As well as a test case that would raise the error that we throw!